### PR TITLE
Sponsor name added

### LINKED ISSUE #1426: Enhancement in the UI of Sponsors' screen
**Actual Behaviour**

Currently, we are displaying only the image and the type of sponsors but the name is not displayed. In our case almost all the images name of the sponsor is visible but suppose its not there (which I realised during the generation of GCI app), then the image is not sufficient in itself.

**Expected Behaviour**
We can add the name of the sponsor as well in the description and the can make the UI simpler. 

**Screenshots of the issue**

**Present one:** 
screenshot_20170423-182657

**Suggested one:** 
screenshot_20170423-183111

**Would you like to work on the issue?**
@the-dagger @mananwason What your thoughts on this?

### PR DESCRIPTION
**Pull request by mahikaw**
Fixes #1426

Changes: [Add here what changes were made in this issue and if possible provide links.]
Sponsor names have also been added in addition to image and sponsor type.
Screenshots for the change: 

device-2017-05-08-225738

### CODE REVIEW

File: android/app/src/main/res/layout/item_sponsor.xml [hunk 1]
```diff
@@ -18,6 +18,17 @@
             android:layout_margin="@dimen/layout_margin_small"
             android:contentDescription="@string/sponsor_image" />
 
+
+        <TextView
+            android:id="@+id/sponsor_name"
+            android:layout_width="79dp"
```
Comment (the-dagger) [inline android/app/src/main/res/layout/item_sponsor.xml]: Why a fixed size?
I think this is the reason as to why the Sponsor name isn't properly centralized for most of the sponsors.

File: android/app/src/main/res/layout/item_sponsor_type.xml [hunk 2]
```diff
@@ -21,4 +21,5 @@
         android:textStyle="bold|italic"
         tools:text="Sponsor" />
 
+
```
Comment (the-dagger) [inline android/app/src/main/res/layout/item_sponsor_type.xml]: Avoid adding empty lines

File: android/app/src/main/res/layout/item_sponsor.xml [hunk 3]
```diff
@@ -27,6 +38,8 @@
             android:textSize="@dimen/text_size"
             android:textStyle="bold|italic"
             tools:text="Sponsor Type" />
+
+
```
Comment (the-dagger) [inline android/app/src/main/res/layout/item_sponsor.xml]: Same.

**Comment by iamareebjamal** [comment]
@mahikaw Also, please squash the commits

**Comment by mahikaw** [comment]
@iamareebjamal I'm not sure how to squash on GitHub since I made the final changes like removing empty lines on GitHub only. Maybe, @the-dagger could squash them while merging. :)

**Comment by free4murad** [comment]
@mahikaw 
Here read this
https://github.com/free4murad/Git-Guide

**Comment by the-dagger** [comment]
@mahikaw 
I can squash them while merging, but it's a good practice to squash it beforehand.

https://ariejan.net/2011/07/05/git-squash-your-latests-commits-into-one/

**Comment by mahikaw** [comment]
@the-dagger @free4murad I am aware off this and I have squashed commits that I changed locally, however when I edit on GitHub itself, and commit online, those commits don't appear in git log and so I am not sure how can I squash them. Could you help me with that?

**Comment by the-dagger** [comment]
@mahikaw Could you check and see if the sponsor names are centralized now?

**Comment by iamareebjamal** [comment]
@mahikaw You have to pull the topic branch locally first, then they will be visible for edits

**Comment by arp95** [comment]
@mahikaw 
apart from that can you please follow the commit guidelines.
Thanks

**Comment by mahikaw** [comment]
@the-dagger I have incorporated the changes requested and squashed the commits. Here is the screenshot of the final changes. Please review. Thanks.

device-2017-05-09-133818

File: android/app/src/main/java/org/fossasia/openevent/adapters/SponsorsListAdapter.java [hunk 4]
```diff
@@ -203,6 +204,9 @@ public int getItemViewType(int position) {
         @BindView(R.id.sponsor_type)
         TextView sponsorType;
 
+        @BindView(R.id.sponsor_name)
```
Comment (arp95) [inline android/app/src/main/java/org/fossasia/openevent/adapters/SponsorsListAdapter.java]: Please resolve the codacy issue.
Rest i think is fine.

Comment (the-dagger) [inline android/app/src/main/java/org/fossasia/openevent/adapters/SponsorsListAdapter.java]: @arp95 
I think it's ok, codacy gives this issue with butterknife.
This isn't an issue as such.

**Pull request merged**